www-src/app/replicator/main.coffee

```diff
@@ -523,26 +520,3 @@ module.exports = class Replicator extends Backbone.Model
     stopRealtime: =>
         @replicationLauncher?.stop()
         delete @replicationLauncher
-
-    # Update cache files with outdated revisions. Called while backup<
-    syncCache:  (callback) ->
```

Comment:
I don't understand.

I test without wifi and I come back with wifi all pictures are uploaded ;)